<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clueDialog</class>
 <widget class="QDialog" name="clueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Clue Report</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>141</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>151</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="instructionsField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>551</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>551</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clue Report</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="clueNumberField">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clue #</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>661</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="callsignField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REPORTED BY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>59</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RADIO LOCATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="radioLocField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>6</y>
      <width>211</width>
      <height>63</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_departing">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>COLLECT  [F1]</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_completed_assignment">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>DISREGARD  [F3]</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_at_ic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>HOLD POSITION  [F4]</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_starting_assignment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>MARK &amp;&amp; LEAVE  [F2]</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clueQuickTextUndoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>UNDO  [Ctrl+Z]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="clueReportPrintCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Print a Clue Report Form as soon as this clue is saved</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_radio_check">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>PROTECT THE CLUE  [F5]</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="descriptionField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cluequick_text_button_welfare_check">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>STANDBY [F6]</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_4</zorder>
  <zorder>buttonBox</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>instructionsField</zorder>
  <zorder>locationField</zorder>
  <zorder>label_2</zorder>
  <zorder>clueNumberField</zorder>
  <zorder>label_6</zorder>
  <zorder>cluequick_text_button_departing</zorder>
  <zorder>cluequick_text_button_completed_assignment</zorder>
  <zorder>cluequick_text_button_at_ic</zorder>
  <zorder>cluequick_text_button_starting_assignment</zorder>
  <zorder>clueQuickTextUndoButton</zorder>
  <zorder>clueReportPrintCheckBox</zorder>
  <zorder>cluequick_text_button_radio_check</zorder>
  <zorder>descriptionField</zorder>
  <zorder>cluequick_text_button_welfare_check</zorder>
  <zorder>line</zorder>
 </widget>
 <tabstops>
  <tabstop>clueNumberField</tabstop>
  <tabstop>descriptionField</tabstop>
  <tabstop>locationField</tabstop>
  <tabstop>instructionsField</tabstop>
  <tabstop>clueReportPrintCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>clueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>clueDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_departing</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_starting_assignment</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_completed_assignment</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_at_ic</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clueQuickTextUndoButton</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_radio_check</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cluequick_text_button_welfare_check</sender>
   <signal>clicked()</signal>
   <receiver>clueDialog</receiver>
   <slot>clueQuickTextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clueQuickTextAction()</slot>
  <slot>clueQuickTextUndo()</slot>
 </slots>
</ui>
